all-in-one/grafana/provisioning/dashboards/CallFlow_Dashboard.json

```
@@ -190,7 +190,7 @@
             "uid": "loki"
           },
           "editorMode": "code",
-          "expr": "{job=\"heplify-server\", src_ip=~\"$src_ip\", dst_ip=~\"$dst_ip\"} | response=~\"$method\" or type!=\"sip\" |~ `${search:raw}` | regexp \"Call-ID:\\s+(?<callid>.+?\r\n)\"",
+          "expr": "{job=\"heplify-server\", src_ip=~\"$src_ip\", dst_ip=~\"$dst_ip\"} | response=~\"$method\" or type!=\"sip\" |~ `${search:raw}` | regexp \"Call-ID:\\s+(?<callid>.+?)\"",
```

This is a step in the right direction but we need to make sure the match stops at a newline for this to work

---

in that case, @tony1661 I believe this is what you're looking for and fixes the issue for me:
```suggestion
          "expr": "{job=\"heplify-server\", src_ip=~\"$src_ip\", dst_ip=~\"$dst_ip\"} | response=~\"$method\" or type!=\"sip\" |~ `${search:raw}` | regexp \"Call-ID:\\s+(?<callid>.+?\\r\\n)\"",
```